be/src/exec/pipeline/adaptive/collect_stats_context.cpp

```diff
@@ -88,11 +88,12 @@ PassthroughState::PassthroughState(CollectStatsContext* const ctx)
           _unpluging_per_driver_seq(ctx->_max_dop) {}
 
 bool PassthroughState::need_input(int32_t driver_seq) const {
-    return _in_chunk_queue_per_driver_seq[driver_seq].size_approx() < MAX_PASSTHROUGH_CHUNKS_PER_DRIVER_SEQ;
+    return _in_chunk_queue_per_driver_seq[driver_seq].queue.size_approx() < MAX_PASSTHROUGH_CHUNKS_PER_DRIVER_SEQ;
 }
 
 Status PassthroughState::push_chunk(int32_t driver_seq, ChunkPtr chunk) {
-    _in_chunk_queue_per_driver_seq[driver_seq].enqueue(std::move(chunk));
+    auto& [chunk_queue, token] = _in_chunk_queue_per_driver_seq[driver_seq];
+    chunk_queue.enqueue(token, std::move(chunk));
```

Comment:
maybe we need check the return status from enqueue?

Reply:
Yes. Fixed.